apps/dashboard/components/side-bar.tsx

```diff
@@ -5,14 +5,21 @@ import List from "@mui/joy/List";
 import ListItem from "@mui/joy/ListItem";
 import ListItemButton, { listItemButtonClasses } from "@mui/joy/ListItemButton";
 import Sheet from "@mui/joy/Sheet";
-import Divider from "@mui/material/Divider"; // Import Divider
+import Divider from "@mui/material/Divider";
 import Link from "next/link";
-import { closeSidebar } from "./utils";
 import ReceiptLongIcon from "@mui/icons-material/ReceiptLong";
-import HomeIcon from "@mui/icons-material/Home";
+import HomeOutlinedIcon from "@mui/icons-material/HomeOutlined";
+import { Typography } from "@mui/joy";
+import { usePathname } from "next/navigation";
+
 import Logo from "./logo";
+import { closeSidebar } from "./utils";
+
+import { getTitle } from "@/app/url";
 
 export default function Sidebar() {
+  const path = usePathname();
+  const isCurrentPath = (href: string) => path === href;
```

Comment:
if path is only used in `isCurrentPath` then change path with `usePathname`

Reply:
Hooks cannot be used inside regular functions.